use-cases/vehicle-routing/src/main/java/org/acme/vehiclerouting/domain/Depot.java

```diff
@@ -0,0 +1,31 @@
+package org.acme.vehiclerouting.domain;
+
+import com.fasterxml.jackson.annotation.JsonIdentityInfo;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.ObjectIdGenerators;
+
+@JsonIdentityInfo(scope = Depot.class, generator = ObjectIdGenerators.PropertyGenerator.class, property = "id")
+public class Depot {
```

Comment:
I'm wondering... why not `record`?

Reply:
Tbh, I did not think about it since I just carried it from the original quickstart, but it would make sense, let me try it and test.

Reply:
As agreed on chat, let's not make this change now, as it causes JSON deserialization problem:
```
Caused by: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No fallback setter/field defined for creator property 'id' (through reference chain: org.acme.vehiclerouting.domain.VehicleRoutePlan["depots"]->java.util.ArrayList[0])
```